study.md

```diff
@@ -43,7 +43,7 @@ Although both Array and Enumerable have a `count` method, Array also defines the
 method `length`.  Why is `length` sensibly defined on Array but not Enumerable?
 
 ```md
-<!-- your answer here -->
+<!-- Enumerable can be infinite? Not very clear about enumerable... I understand that if I use a range like (1..10) I can't applied .length on this one because .length is a method that acts on groups of elements, it doesn't act on each of the items to get the length. Instead .count is a method that goes through each item in the selection or group. ... -->
```

Comment:
Yes, that's one reason why `length` isn't applicable on `Enumerable`s.

`length` makes sense on `Array` because arrays have a finite length.  It does not make sense for an `Enumerable` to have a `length` because it might not be finite.  For example, an enumerable that returns a digit of π on each iteration does not have a finite length.